client/src/pages/Home/index.js

```diff
@@ -1,24 +1,65 @@
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
❓ Why was this import added? It's not explicitly used anywhere, and [the "new" transform][1] means this is no longer required in JSX files generally.

[1]: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html